Makefile

```diff
@@ -0,0 +1,33 @@
+PREFIX ?= /usr/local
+BASH_COMPLETION_DIR := $(shell pkg-config --variable=completionsdir bash-completion) 
```

Comment:
I don't have modern OS X, but this doesn't fail on my snow leopard server, so I can't reproduce. Any ideas?